### ISSUE TITLE
Infinite Scroll by User ID

### ISSUE THREAD

**Issue by pottstarr** [DDKnoll/Instagram-Infinite-Scroll#14]
Hi, is there a way to load the user id infinitely as well? I noticed it only loads a few/most recent. Thanks in advance!

**Comment by cooldept** [comment]
Yeah, I've been trying to find a way to do that for the past few hours, but my JS skills are not advanced at all and there is not much documentation for the novice. 
This is the best JS integration of instagram I've seen around btw!!

**Comment by cooldept** [comment]
Phew I figured it out. I took me about 5 hours (no kidding) of looking in the wrong places...
It's a tag parameter you need to change (at least in cases where you are using the USER mode).

The parameter is in file instagram.js and you need to search and replace this tag:
     next_max_tag_id
with this: 
    next_max_id

At least that's what worked for me, but I tested so many things I have idea of the code is even close to the original anymore... 

But this I found out about from: https://instagram.com/developer/endpoints/
So maybe Instagram recently changed this, or maybe in the "user" search mode this play a role..
I hope this helps anyone...
